docs/microbit_micropython_api.rst

```diff
@@ -266,6 +321,27 @@ There is an I2C bus on the micro:bit that is exposed via the `i2c` object.  It h
     # write buf to device with addr; repeat=True means a stop bit won't be sent.
     i2c.write(addr, buf, repeat=False)
 
+.. _sound
+
+Sounds **V2**
+------
+
+A set of expressive sounds are available to the micro:bit **V2** and can be
```

Comment:
Two "and" in the same sentence.
```suggestion
A set of expressive sounds are available to the micro:bit **V2**, can be
```

Reply:
Reading it aloud. I wondered if a full stop might be better?
```suggestion
A set of expressive sounds are available to the micro:bit **V2**. They can be
```

Reply:
Yeah, that works better.